app/main/posts/views/post-filters.service.js

```diff
@@ -30,10 +30,28 @@ function PostFiltersService(_, FormEndpoint, TagEndpoint, $q) {
         getModeId: getModeId,
         getModeEntity: getModeEntity,
         countFilters: countFilters,
+        cleanDeprecatedValuesFromSavedSearch: cleanDeprecatedValuesFromSavedSearch,
         reactiveFilters: true,
         qEnabled: false
     };
 
+
+    /**
+     * Looks for keys that are NOT present in currentFilters but that are in the savedSearch filters (which means they are removed)
+     * and removes them from the saved search filters array.
+     * Does not handle removal where the key exists but the values are an array and some are missing. Need to fix that.
+     **/
+    function cleanDeprecatedValuesFromSavedSearch(currentFilters, savedSearch) {
```

Comment:
maybe just cleanRemovedValuesFromSavedSearch? Deprecated seems like the wrong term, but I'm not sure whats better

Reply:
yea that's a little better. Still weird.. but better. :) thanks